app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -702,7 +700,7 @@ class TabTrayView(
             layoutManager?.scrollToPosition(recyclerViewIndex)
             smoothScrollBy(
                 0,
-                - resources.getDimensionPixelSize(R.dimen.tab_tray_tab_item_height) / 2
```

Comment:
I don't think we want to remove this / 2 either because that's removing the offset of showing a bit of the tab item above.

Reply:
You're right. 👍